lib/axe/api/results/rule.rb

```diff
@@ -15,21 +15,31 @@ class Rule < ValueObject
           attribute :nodes, ::Array[CheckedNode]
         end
 
-        def failure_message(index)
+        def failure_messages(index)
           [
-            help_message(index+1),
-            node_count_message
-          ].concat(nodes.map(&:failure_message))
+            title_message(index+1),
+            *[
+              helpUrl,
+              node_count_message,
+              "",
+              nodes.map(&:failure_messages).map{|n| n.push("")}.flatten.map(&indent)
+
+            ].flatten.map(&indent)
+          ]
         end
 
         private
 
-        def help_message(count)
-          "#{count}) #{help}: #{helpUrl}\n"
+        def indent
+          -> (line) { line.prepend(" " * 4) }
```

Comment:
@jasonkarns This throws an error when `line` is `nil`. I came across the issue while trying to replicate issues @dylanb is having running the examples in the attest-ruby repo. Specifically the rspec example.